limiter.go

```diff
@@ -96,8 +100,9 @@ func (r *RateLimiter) EvaluateRequest(incomingReq Request) (evalError error) {
 		userBucket = &Bucket{
 			UserIdentifier: incomingReq.UserIdentifier,
 			FillRate:       r.cfg.FillRate,
-			Tokens:         r.cfg.BucketCapacity,
+			Tokens:         0,
```

Comment:
Initializes bucket to 0 as the evaluation on first check will auto fill it to `BucketCapacity` volume